logger/log_test.go

```diff
@@ -210,10 +205,7 @@ func TestFileLoggerSizeLimit(t *testing.T) {
 		t.Fatalf("Should be statement about rotated log and backup name, got %s", content)
 	}
 
-	// Remove all files
-	removeDir(t, tmpDir)
-	tmpDir = createDir(t, "nats-server")
-	defer removeDir(t, tmpDir)
+	tmpDir = t.TempDir()
```

Comment:
So we do not need to clean these up? The directory?

Reply:
yes not needed, the testing framework auto cleans these up when the test is done

Reply:
ok nice! And we have actually test that is the case yes?